tests/generic_config_updater/gu_utils.py

```diff
@@ -40,6 +42,12 @@ def apply_patch(duthost, json_data, dest_file):
         json_data: Source json patch to apply
         dest_file: Destination file on duthost
     """
+    if duthost.is_multi_asic:
+        for operation in json_data:
+            if not operation["path"].startswith("/{}".format(HOST_NAME)) and \
+               not operation["path"].startswith("/{}".format(ASIC_PREFIX)):
+                operation["path"] = "/" + HOST_NAME + operation["path"]
```

Comment:
Do we have UT cover this?

Reply:
This change is in test path, but I will add AAA and other table test for multiasic after this PR checked in: https://github.com/sonic-net/sonic-mgmt/pull/13210